rotkehlchen/chain/bitcoin/__init__.py

```diff
@@ -79,8 +117,8 @@ def _check_blockstream_for_transactions(
 
 
 def _check_blockchaininfo_for_transactions(
-        accounts: List[BTCAddress],
-) -> Dict[BTCAddress, Tuple[bool, FVal]]:
+        accounts: list[BTCAddress],
+) -> dict[BTCAddress, tuple[bool, FVal]]:
```

Comment:
You really really should not touch code unless you need to. This is completely unnecessary. Refer to my post about it in the beginning.